app/views/component/about/_about.html.haml

```diff
@@ -1,10 +1,9 @@
 %div.about-container
   .about-summary
-    %img{ src: asset_path(SETTINGS[:site_logo_alt]), alt: "#{SETTINGS[:site_title]} Logo", class: "logo"}
+    %img{ src: asset_path(SETTINGS[:site_logo_alt]), alt: "#{SETTINGS[:site_title]} large logo", class: "logo", aria: { label: t('branding.logo') } }
```

Comment:
I winder if we should put the whole text that is on the image here: 
![image](https://github.com/user-attachments/assets/cdc786a2-e15c-4f0d-b786-f685cfda87b2)

The translation only returns "Charm care"
![image](https://github.com/user-attachments/assets/aefd99c5-1a25-4a71-9a85-33a2edff37a8)

Reply:
Sounds good - I added the text in the image to the accessible label!